content/auth/basic.textile

```diff
@@ -23,11 +23,15 @@ Basic authentication is the simplest way to authenticate with Ably. It requires
 The following is an example of using basic authentication:
 
 ```[realtime_javascript]
-  var realtime = new Ably.Realtime({ key: '{{API_KEY}}' });
+  const realtime = new Ably.Realtime.Promise({
```

Comment:
There are some caveats to using the `'ably/promise'` import so the recommended way to use promises is via the `.Promise` constructors. Although since we're releasing v2 soon it might actually be better to just follow the new API in the example (ie don't use the `.Promise` constructors and import directly from `'ably'`)

Reply:
I think in the ideal world @ttypic we would use the import as the example. But that would only be displayed on one or two pages out of all of our docs. People may have skipped that part and then end up with friction just getting started with our product. While we're building up to the 2.0 release, I'd like to keep it so we only import ably, and then use the `.Promise` constructor throughout the examples. They'll be easy to search and replace when the need arises too.